Hymns/Settings/SettingsView.swift

```diff
@@ -10,13 +10,12 @@ struct SettingsView: View {
 
     var body: some View {
         VStack(alignment: .leading) {
-            CustomTitle(title: "Settings")
+            Text("Settings").customTitleLayout().padding(.top)
```

Comment:
Not sure, what this is for. The problem is it throws off consistency with browse and favorites